plugins/policy-recommendation/policy_recommendation_job.py

```diff
@@ -1168,7 +1196,7 @@ def main(argv):
             option,
             start_time,
             end_time,
-            ns_allow_list,
+            broadcast_ns_allow_list.value,
```

Comment:
Do we need to check broadcast_ns_allow_list is None? Or it must have field `value` after `spark.sparkContext.broadcast(NAMESPACE_ALLOW_LIST)` is called?

Reply:
Yep `broadcast_ns_allow_list` would not be None here so I skip the check.
For other distribued running map functions like `generate_k8s_np()`, I added the None check to unblock the unit tests.